mmdet/datasets/utils.py

```diff
@@ -67,3 +70,22 @@ def show_ann(coco, img, ann_info):
     plt.axis('off')
     coco.showAnns(ann_info)
     plt.show()
+
+
+def get_dataset(data_cfg):
+    if isinstance(data_cfg['ann_file'], list) or \
+            isinstance(data_cfg['ann_file'], tuple):
```

Comment:
You can use `isinstance(data_cfg['ann_file'], (list, tuple))` to check multiple types.